src/test/scala/mesosphere/marathon/api/validation/V2AppDefinitionValidatorTest.scala

```diff
@@ -23,6 +25,109 @@ class V2AppDefinitionValidatorTest extends MarathonSpec {
     validateJsonSchema(app)
   }
 
+  for (
```

Comment:
Not a big fan of programmatically created test functions - it is harder to find the problem, if it breaks. But that is personal taste.